src/ja/user/config/add.py

```diff
@@ -7,20 +9,73 @@ class AddCommandConfig(UserConfig):
     Config for the add command of the user client.
     """
 
+    def __init__(self, config: UserConfig,
+                 job: Job = None, attach: bool = False, blocking: bool = True):
+        super().__init__(ssh_config=config.ssh_config, verbosity=config.verbosity)
+        self._job = job
+        self._attach = attach
+        self._blocking = blocking
+
     @property
     def job(self) -> Job:
         """!
         @return: The job to be added.
         """
+        return self._job
 
     @property
     def attach(self) -> bool:
```

Comment:
I think that we discussed this option in the meeting and agreed that we would not implement it.

Reply:
After the meeting :)
But yeah I think we can leave this one, it is not crucial to our project.

Reply:
I removed it.